ecosystem/sep-0031.md

```diff
@@ -442,14 +441,23 @@ Name | Type | Description
 
 ##### Success (201 Created)
 
-This is the successful case where a Receiving Anchor confirms that they can fulfill this payment as described. The response body should be a JSON object with the following values
+This is the successful case where a Receiving Anchor either confirms that they can fulfill this payment as described or that confirmation is pending.
+
+Anchors should check the `status` response attribute to determine whether or not the Receiving Anchor is ready to receive the payment. 
+
+If `status` is `pending_sender`, `stellar_account_id`, `stellar_memo`, & `stellar_memo_type` should be present. 
+
+If `status` is `pending_receiver`, the Receiving Anchor is processing the information sent and determining if the transaction can proceed. In this case, `stellar_account_id`, `stellar_memo`, & `stellar_memo_type` will not be present, and the Sending Anchor should monitor the transaction's status until it moves to `pending_sender` or `error`. 
+
+Sending Anchors can monitor the transaction's status by registering a callback URL via [`PUT /transactions/:id/callback`](#put-transaction-callback) or by polling [`GET /transactions/:id`](#get-transaction). When the transaction status moves to `pending_sender`, the Stellar account & memo fields should be populated.
 
 Name | Type | Description
 -----|------|------------
 `id` | string | The persistent identifier to check the status of this payment transaction.
-`stellar_account_id` | string | The Stellar account to send payment to.
-`stellar_memo_type` | string | The type of memo to attach to the Stellar payment (`text`, `hash`, or `id`).
-`stellar_memo` | string | The memo to attach to the Stellar payment.
+`stellar_account_id` | string | (optional) The Stellar account to send payment to.
+`stellar_memo_type` | string | (optional) The type of memo to attach to the Stellar payment (`text`, `hash`, or `id`).
+`stellar_memo` | string | (optional) The memo to attach to the Stellar payment.
+`status` | string | (optional) The initial status of the transaction, either `pending_sender` or `pending_receiver`.
```

Comment:
Since we're making a breaking change and going to v2.0.0, what do you think of changing the response body of `POST /transactions` to the same response body of `GET /transactions/:id`?

Reply:
I think that makes a lot of sense. We’ll have to be intentional about adding more guidance in the SEP and documentation though so it’s clear which attributes are useful immediately after transaction creation.